### PR TITLE
chore: CONTRIBUTING is out-of-date

### PR DESCRIPTION
**Pull request by PartMan7**
All Submissions:

* [ ] Have you followed the guidelines in our [Contributing document](https://github.com/lost-pixel/lost-pixel/blob/main/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/lost-pixel/lost-pixel/pulls) for the same update/change?

It's a bit hard to follow the guidelines in CONTRIBUTING.md when I'm changing them 'o.o

Resolves: #284 

Wasn't sure which branch should've been the new base, please let me know if it's a different one.

### CODE REVIEW

**Comment by d-ivashchuk** [comment]
hey @PartMan7! I was using the great template from vercel for contributing doc, but we did not spend time to adhere to the same contributing principles. so far you could base anything that you want to introduce to lost pixel from main & do PRs against main!

**Comment by PartMan7** [comment]
> hey @PartMan7! I was using the great template from vercel for contributing doc, but we did not spend time to adhere to the same contributing principles. so far you could base anything that you want to introduce to lost pixel from main & do PRs against main!

I see, thanks!
